packages/cli/src/util/postHog/__test__/postHog.test.ts

```diff
@@ -0,0 +1,108 @@
+import { capture } from "..";
+import * as postHogClient from "../client";
+import * as moduleManifestUtil from "../../moduleManifestUtil";
+import * as config from "../../config";
+
+describe("postHog", () => {
+  beforeEach(() => {
+    // jest.resetAllMocks();
+  });
+
+  afterEach(() => {
+    delete process.env.MM_DEV;
+  });
+
+  it("should call capture on the postHog client", () => {
+    const mockPostHogClient = { capture: jest.fn() };
+
+    jest
+      .spyOn(postHogClient, "postHogClient")
+      // @ts-ignore
+      .mockImplementation(() => mockPostHogClient);
+
+    capture({
+      distinctId: "abc",
+      event: "ate pizza",
+    });
+
+    expect(mockPostHogClient.capture).toHaveBeenCalledWith({
+      distinctId: "abc",
+      event: "ate pizza",
+    });
+  });
+
+  it("should call capture on the postHog client - with config.anonymousId if options.distinctId is blank", () => {
+    // jest.spyOn(moduleManifestUtil, "getConfig").mockImplementation(() => {
```

Comment:
@psugihara if you have a sec, when I uncomment this, it makes this test pass but all others fail.  i think i have to clear mocks before the other tests, but that doesn't fix it.  i will take a look tomorrow unless you know what I should do

Reply:
think it needed mockRestore because you had changed the implementation 🤷